.travis.yml

```diff
@@ -9,7 +10,6 @@ install:
   - echo set -e >> ./travis.sh
   - echo set -x >> ./travis.sh
   - echo set -o pipefail >> ./travis.sh
-  - echo echo Running on Mono 3.x >> ./travis.sh

```

Comment:
maybe just change this to `echo mono --version >> ./travis.sh` to get the mono version up to date.